src/components/QuranReader/SidebarNavigation/SidebarNavigation.tsx

```diff
@@ -104,7 +105,7 @@ const SidebarNavigation = () => {
 };
 
 const TABLET_WIDTH = 768;
-const isMobile = () => {
+export const isMobile = () => {
```

Comment:
I think we should take this out to a util since the logic is not related to `SidebarNavigation`.